src/module.h

```diff
@@ -10,6 +10,7 @@
 #include "tsdb.h"
 
 extern RedisModuleType *SeriesType;
+extern RedisModuleCtx *rts_staticCtx;
```

Comment:
`module.c` includes `module.h` "defining" `rts_staticCtx` twice: as an `extern` than locally

Reply:
@swilly22 In module.c it the definition and in module.h is the declaration.
 extern means declaring without definition.
 Meir in libmr doing the same thing with RedisModuleCtx* mr_staticCtx;